src/main/java/com/byndyusoft/template/client/configuration/ClientProperties.java

```diff
@@ -0,0 +1,22 @@
+package com.byndyusoft.template.client.configuration;
+
+import java.time.Duration;
+import javax.validation.constraints.NotNull;
+import lombok.Data;
+import org.apache.http.impl.conn.PoolingHttpClientConnectionManager;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.NestedConfigurationProperty;
+import org.springframework.validation.annotation.Validated;
+
+@Data
+@Validated
+@ConfigurationProperties(prefix = "clients.templateClient")
+public class ClientProperties {
+  @NestedConfigurationProperty
+  private PoolingHttpClientConnectionManager connectionManager;
```

Comment:
_Vkusovshina_. But it seems to me that it is better if this field is the last one, and the baseUrl is the first one.****

Reply:
done